edb/lib/math.edgeql

```diff
@@ -28,50 +28,50 @@ math::abs(x: std::anyreal) -> std::anyreal
 
 
 CREATE FUNCTION
-math::ceil(x: std::int64) -> std::float64
+math::ceil(x: std::int64) -> std::int64
 {
     SET volatility := 'IMMUTABLE';
-    USING SQL FUNCTION 'ceil';
+    USING SQL FUNCTION 'edgedb.ceil';
 };
 
 
 CREATE FUNCTION
 math::ceil(x: std::float64) -> std::float64
```

Comment:
Why you didn't change this one to return bigint?

Reply:
Why? `math::ceil(1e300)` should be `1e300`, not a bigint (which would also be slow).

Reply:
1e300 is an imprecise float, ceiling it shouldn't suddenly yield a precise result. IMO float operations and functions have nothing to do with decimals/bigints.

Reply:
That's what python3 does:
```
>>> type(math.ceil(1e300))
<class 'int'>
```
Python2 returned a float. So I assume that there was some reasoning behind that.

Reply:
> So I assume that there was some reasoning behind that.

Mostly historical.  It's likely that pre-Python 3 `math.ceil` predates the `long` type.